templates/absences/add.html.twig

```diff
@@ -3,7 +3,7 @@
 {% extends 'base.html.twig' %}
 
 {% block specificjs %}
-  <script type='text/JavaScript' src='{{ app.request.getBaseURL() }}/absences/js/modif.js?version={{ version }}'></script>
+  <script type='text/JavaScript' src='{{ app.request.getSchemeAndHttpHost() }}/absences/js/modif.js?version={{ version }}'></script>
 {% endblock %}
```

Comment:
You should use "asset" : 
<pre>
<script type='text/JavaScript' src='{{ asset("absences/js/modif.js") }}?version={{ version }}'></script>
</pre>